src/test/java/junit/samples/ListTest.java

```diff
@@ -61,7 +61,7 @@ public void testRemoveAll() {
     }
 
     public void testRemoveElement() {
-        fullList.remove(3);
-        assertTrue(!fullList.contains(3));
+        fullList.remove(Integer.valueOf(3));
+        assertTrue(!fullList.contains((Integer) 3));
```

Comment:
No need to cast here (the test doesn't cast in other calls to contains)

Also while you are here, please replace every place that does `assertTrue(!expression)` with `assertFalse(expression)`